docs/UserGuide.md

```diff
@@ -144,25 +144,40 @@ Examples:
 * `find n/alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find n/alex david'](images/findAlexDavidResult.png)
 
-### Mailing student(s) : `mail`
+### Mailing a particular student based on index : `mail-index`
 
-Mails a person from the student roster.
+Mails a person from the student roster based on the index number shown in the application.
 
-Format: `mail [e/EMAIL]…​`
+Format: `mail-index INDEX`
 
 * This opens the default email application on the system with the specified mail address pre-filled at the receiver's
   address.
-* This can be used to specify multiple addresses to send the same mail in a single go as well.
 
 Example:
-* `mail e/johndoe@example.com` would open the default mail on the system with the "to" box filled with the
-  specified mail. Now, the email is ready to be sent to `johndoe@example.com`.
+* `mail-index 2` would open the default mail on the system with the "to" box filled with the
+  specified mail. Now, the email is ready to be sent to the person with the index 2 as shown on the application.
 
-### Bulk Emails to everyone on the database : `mailall`
+### Mailing student subset based on arguments : `mail-x`
+
+Mails a subset of students from the contact list. This is a broader version of mail functionality as it can
+cover multiple students at once in order to mail them simultaneously.
+
+Format: `mail-x [e/EMAIL] [g/GROUP] [m/MOD] [n/NAME]`
+
+* This opens the default email application on the system with all the mail addresses specified by the arguments.
+* The mail addresses are additively added to the receiver's box.
```

Comment:
Perhaps instead of saying "additively", a better phrase to use would be something like
"Anyone who matches at least one of the specified arguments will be included in the mailing list"

Reply:
Okay, that sounds good!